src/resources/Platforn/Platform.ts

```diff
@@ -0,0 +1,31 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ComponentVersion, CrawlingModule, MaestroVersionOptions, UpdateStatus} from './PlatformInterfaces';
+
+export default class Platform extends Resource {
```

Comment:
I think it would be more representative if the resource was called `CrawlingModule` what do you think?

Reply:
That makes sense. At first it looked like the resource was named after the services on the platform.